webui-vue/public/404.html

```
@@ -0,0 +1,12 @@
+<html>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)